exercises/156-findPairForSum/test.js

```diff
@@ -9,7 +9,7 @@ test('the function must find a pair of numbers that add up to the goal number.',
     const findPairForSum = rewire("./app.js").__get__("findPairForSum");
 
     expect(findPairForSum([3, 34, 4, 12, 5, 2], 9)).toEqual([4, 5])
-    expect(findPairForSum([1, 2, 3, 4, 5], 7)).toBe([4, 3])
```

Comment:
What if 4,3 is mathcing? your test should be compatible with both approaches or change the input to reduce the amount of possible outputs to 1